README.md

```diff
@@ -49,35 +76,37 @@ For any method that modifies chain state, the key to sign with can be controlled
 1. Vault Signing:
    By setting `VAULT_URL` and `VAULT_SECRET`an external [Vault](https://www.vaultproject.io/) instance will be used to sign transactions. The URL should point to a transit engine in Vault that has Ed25519 keys in it. To refer to a key when signing use the Vault name and version `${name}-${version}` e.g. `alice-1`
 
-## Webhooks (alpha)
+### Authentication
+
+The REST API uses [passport.js](https://www.passportjs.org/) for authentication. This allows the service to be configurable with multiple strategies.
+
+Currently there are two strategies available:
+
+1. Api Key:
+   By configuring `apiKey` as a strategy, any request with the header `x-api-key` will be authenticated with this strategy. The env `API_KEYS` can be used to provide initial keys
+1. Open:
```

Comment:
```suggestion
2. Open:
```

Reply:
This is markdown so it increments on display. This way makes it easy to move points without messing up the numbering.
<img width="876" alt="Screen Shot 2022-10-17 at 8 45 36 AM" src="https://user-images.githubusercontent.com/86971143/196180480-9963cae9-c016-4849-8ce1-515e3ef51690.png">